portal/src/components/DownloadAsset/DownloadAsset.scss

```
@@ -0,0 +1,40 @@
+@import "~@fremtind/jkl-core/variables/_all.scss";
+
+.jkl-portal-downloadasset {
+    display: inline-block;
+    padding: rem(16px);
+
+    &--dark-bg {
+        background-color: black;
+    }
+    &--image {
+        width: rem(300px);
+        height: auto;
+        &:hover {
+            cursor: pointer;
+        }
+    }
+    &--name {
+        padding: rem(16px);
+        display: block;
+        &-dark {
```

```suggestion
        &--dark {
```

Evt. endre dette basert på `.jkl-portal-downloadasset--darkbg` i stedet for å sette eget klassenavn her.